etc/config/c++.amazon.properties

```diff
@@ -2867,6 +2873,10 @@ compiler.nvcxx_arm_cxx23_9.demangler=/opt/compiler-explorer/hpc_sdk/Linux_aarch6
 compiler.nvcxx_arm_cxx23_9.exe=/opt/compiler-explorer/hpc_sdk/Linux_aarch64/23.9/compilers/bin/nvc++
 compiler.nvcxx_arm_cxx23_9.semver=23.9
 
+compiler.nvcxx_arm_cxx23_11.demangler=/opt/compiler-explorer/hpc_sdk/Linux_aarch64/23.11/compilers/bin/nvdecode
```

Comment:
I forget what's going on here... but none of these `aarch64` compilers are installed here:
```
Nov/21 04:02 admin-node~ $ ls /opt/compiler-explorer/hpc_sdk/
Linux_aarch64  Linux_x86_64
Nov/21 04:02 admin-node~ $ ls /opt/compiler-explorer/hpc_sdk/Linux_aarch64/
Nov/21 04:02 admin-node~ $ 
Nov/21 03:37 admin-node~ 1.2s $ /opt/compiler-explorer/hpc_sdk/Linux_aarch64/23.11/compilers/bin/nvc++
/opt/compiler-explorer/hpc_sdk/Linux_aarch64/23.11/compilers/bin/nvc++: command not found
````

...but that seems to be repeated above in the others so that's nothing new. I'm confused but happy to merge as is and assume somehow y'all know what you're doing here...

Reply:
I suspect that the aarch64 compilers are not cross compilers but native ones, and thus the installer decided not to install them on amd64

Reply:
Right, the aarch64 compilers are not cross-compilers. They need to run on aarch64 (or on x86 under qemu-user).

Reply:
Got it. I wonder why they're here? Hopeful for the future I guess :)